docs/components/best-practices/architecture/deciding-about-your-stack.md

```diff
@@ -9,83 +9,77 @@ tags:
   - Maven
 ---
 
-If you don't have special architecture requirements, we recommend using SaaS following the proposed greenfield stack.
+This document provides guidance on selecting the appropriate technology stack for your Camunda 8 project, especially if you don't have special architecture requirements. We recommend using SaaS for a streamlined experience.
 
 :::caution Camunda 8
-This best practice targets Camunda 8 only! If you look for Camunda 7, please refer to [Deciding about your Camunda 7 stack](../deciding-about-your-stack-c7/).
+This best practice specifically targets Camunda 8. For Camunda 7, please refer to [Deciding about your Camunda 7 stack](../deciding-about-your-stack-c7/).
 :::
 
 ## The greenfield stack
 
-We like to give one greenfield stack recommendation, which is the stack you can simply use if there is no reason against it. And while we went through long and detailed discussions to come to this recommendation, it _doesn't_ mean that it is necessarily superior to alternative stacks. You can still feel confident if you go down another route (note below for alternative options).
-
-The stack looks pretty similar in various programming language. Please use the programming language your team is most familiar with. If in doubt, use Java or JavaScript.
+Our greenfield stack recommendation is a result of extensive discussions and evaluations. While not the only option, it is a solid choice if there are no specific reasons to choose an alternative. Your choice of programming language should align with your team's expertise; we suggest Java or JavaScript for their broad applicability and support.
 
 ### The Java greenfield stack
 
 ![greenfield stack architecture diagram](deciding-about-your-stack-assets/greenfield-architecture.png)
 
-Use the following stack:
+For a Java-based setup using Camunda 8 SaaS and Spring Boot, use the following stack:
 
-1. Use [Camunda 8 SaaS](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral) and create a cluster there
+1. **Camunda 8 SaaS:**
 
-1. Build your process solution project as a [Spring Boot](https://spring.io/projects/spring-boot) application, using the [Spring Zeebe](/docs/apis-tools/community-clients/spring.md).
+   1. **Sign up:** Users need to sign up for Camunda 8 SaaS. This can be done through the [Camunda SaaS registration page.](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral)
+   2. **Create a cluster:** After signing up, users should create a cluster. They can follow the guide on [Creating a Cluster in Camunda 8](/guides/create-cluster.md) which provides step-by-step instructions on setting up a new cluster in the Camunda 8 environment.
```

Comment:
Since this is a Best Practices page, could we assume users already have an account and a cluster? I would like to drive everyone through our [Getting Started Guide content](https://docs.camunda.io/docs/guides/#getting-started) first.

Reply:
Something more like?

1) If you're new to Camunda SaaS, please check out our Getting Started Guide to set up your environment.

2. **Create a cluster:** After signing up, users should create a cluster. They can follow the guide on [Creating a Cluster in Camunda 8](/guides/create-cluster.md) which provides step-by-step instructions on setting up a new cluster in the Camunda 8 environment.